README.md

```diff
@@ -56,23 +56,25 @@ var validator = hannibal.create({
 });
 
 // Check a valid user
-validator({
+var basicRslt1 = basicValidator({
     name: "John Smith",
     age: 53,
     address: {
         street: "The underground",
         city: "Los Angeles"
     }
-}) // {isValid: true, data: {name: "John Smith", ...}}
+})
+assert(basicRslt1.isValid);
 
 // Check an invalid user
-validator({
+var basicRslt2 = basicValidator({
     name: "Templeton Peck",
-    age: "38",
+    age: "foo",
     address: {
         city: "Los Angeles"
     }
-}) // {isValid: false, error: {age: {type: "is not a string", address: {street: {required: "is not provided"}}}}, data: {name: "Templeton Peck"}}
```

Comment:
We could do a [assert.deepEqual](https://nodejs.org/api/assert.html#assert_assert_deepequal_actual_expected_message) on the `error` message, thoughts?